### PR TITLE
Implement jeopardy game

### PR DESCRIPTION
**Pull request by bsrecker**
Implement jeopardy game

### CODE REVIEW

File: src/cogs/jeopardy.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+
+class Jeopardy(commands.Cog):
+    """A Jeopardy question answer game utilizing Levenshtein Distance to calculate answer matching (beta)."""
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.attempts = 0
+
+    @commands.max_concurrency(1, per=commands.BucketType.guild, wait=False)
+    @commands.command()
```
Comment (mudkipdev) [inline src/cogs/jeopardy.py:18]: Add a description, maybe?
`@commands.command(help = "Generates a Jeopardy question from over 200,000 historical Jeopardy Questions.")`

File: src/cogs/jeopardy.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    async def jeopardy(self, ctx):
+        """Generates a Jeopardy question from over 200,000 historical Jeopardy Questions."""
+        try:
+            with open('JEOPARDY_QUESTIONS1.json') as f:
+                data = json.loads(f.read())
+                position = random.randint(0, 199999)
+                question = data[position]['question']
+                answer = data[position]['answer']
```
Comment (mudkipdev) [inline src/cogs/jeopardy.py:26]: It's already pretty readable without all these lines, maybe merge them into one or two

File: src/cogs/jeopardy.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.attempts = 0
+
+    @commands.max_concurrency(1, per=commands.BucketType.guild, wait=False)
+    @commands.command()
+    async def jeopardy(self, ctx):
+        """Generates a Jeopardy question from over 200,000 historical Jeopardy Questions."""
+        try:
+            with open('JEOPARDY_QUESTIONS1.json') as f:
```
Comment (mudkipdev) [inline src/cogs/jeopardy.py:22]: Should we hardcode this?

File: src/cogs/jeopardy.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                data = json.loads(f.read())
+                position = random.randint(0, 199999)
+                question = data[position]['question']
+                answer = data[position]['answer']
+
+                category = data[position]['category']
+                value = data[position]['value']
+                airDate = data[position]['air_date']
```
Comment (mudkipdev) [inline src/cogs/jeopardy.py]: camel case lol

File: src/cogs/jeopardy.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+                em = discord.Embed(color=config["game_color"])
+                em.title = "Jeopardy!"
+                em.add_field(name='Question', value=str(question))
```
Comment (mudkipdev) [inline src/cogs/jeopardy.py]: Is it not already a string?

**Review by mudkipdev — CHANGES_REQUESTED** [review]
Looks pretty good, read my comments though

File: src/cogs/jeopardy.py (hunk 1 above)
Comment (bsrecker) [reply src/cogs/jeopardy.py:18]: This should be handled by the docstings already.

File: src/cogs/jeopardy.py (hunk 5 above)
Comment (bsrecker) [reply src/cogs/jeopardy.py]: Resolved

File: src/cogs/jeopardy.py (hunk 4 above)
Comment (bsrecker) [reply src/cogs/jeopardy.py]: Resolved

File: src/cogs/jeopardy.py (hunk 3 above)
Comment (bsrecker) [reply src/cogs/jeopardy.py:22]: Unless there is a different way you wish to do this I say we leave it like this for now.

**Review by mudkipdev — APPROVED** [review]

**Pull request merged**